presto-main/src/test/java/com/facebook/presto/block/TestDictionaryBlock.java

```diff
@@ -41,6 +41,22 @@ public void testSizeInBytes()
         assertEquals(dictionaryBlock.getSizeInBytes(), dictionaryBlock.getDictionary().getSizeInBytes() + (100 * SIZE_OF_INT));
     }
 
+    @Test
+    public void testLogicalSizeInBytes()
+    {
+        // The 10 Slices in the array will be of lengths 0 to 9.
+        Slice[] expectedValues = createExpectedValues(10);
+
+        // The dictionary within the dictionary block is expected to be a VariableWidthBlock of size 95bytes.
```

Comment:
Nit: when I read "95bytes" it jumps out at me as missing a space

We could also write this as 95B, which would be consistent with values like 16MB, but that seems less natural than "95 bytes" (and could be confused with 95 billion).

Reply:
Makes sense. I will fix this in all of my comments.